### ISSUE TITLE
No error logs or debug/verbosity mode

### ISSUE THREAD

**Issue by hashworks** [thedevs-network/kutt#420]
As far as I see, kutt offers no way to log any errors. It won't show any by default and there is no debug/verbosity mode – and if there is one it isn't documented.

For example, I had a working instance and suddenly I'm getting 500 errors on link creation. Opening old links, login, listing old links etc. works fine, only the link creation fails. However, there is no error output whatsoever besides the `{"error":"An error occurred."}` json output.

Basically I have to guess what's wrong, I think it's absolutely required that there is some kind of error logging.

**Comment by poeti8** [comment]
You can see the errors in Node.js console. So I guess what you mean is to send more info of what happened inside API response as well?

**Comment by hashworks** [comment]
There are no errors in my node console – at least not for that specific 500 error.

**Comment by vmwsree** [comment]
I ms tuck . . i mablet to create link.. not able to delete or update.. it shows error occurred.. no way to know whats going on

**Comment by hashworks** [comment]
This is something that should be configurable over a configuration file, parameter or environment variable.

**Comment by ksingh7** [comment]
a parameter in .env to control debuging is much needed.
